### ISSUE TITLE
Videos are no longer added to History section. Error occurred.../cache/PlayerData

### ISSUE THREAD

**Issue by mathkkad** [yuliskov/SmartTube#1109]
Hi!

Since yesterday i am having this error on all my devices on the app's stable versions like v13.47_r or v13.31_r
Some devices were booted up just to test (not used in a while and everything was fine before) and i also uninstalled and reinstalled/cleared data on multiple devices but i am still having the problem

Basically any new video that you play does not go to the History section anymore (on other apps or in PC browser works fine) and i get this error the first time i load a video with a clean cleared cache app:

Error occurred. Detailed info has been saved to /storage/emulated/0/Android/data/com.teamsmart.videoamnager.tv/cache/PlayerData

When i manually go to that folder path on a device that has root it says it's empty

Please help as i use the History section a lot. Thank you

**Comment by Tailszefox** [comment]
Same issue here: videos watched with the app don't appear in the history section, in the app or on YouTube. In fact any video I've watched with the app acts like it hasn't been watched at all. This must be a very recent change since it worked less than 24 hours ago.

For some reason the path the app gives when the error occurs is slightly different: `/storage/emulated/0/Android/data/com.liskovsoft.smarttubetv.beta/cache/PlayerData`. The file does exist but it's very big (1.7Mb of just text) and I worry there's identifying stuff in there. Is there any way to send it directly to the dev instead of uploading it publicly on GitHub?

**Comment by doczong** [comment]
I am sure others will chime in as well, but I am getting the same error.  Seems to be systemic.

**Comment by DTK21** [comment]
Don't know if it just requires the same fix from an update ways back or if it's a YouTube change that caused it. But the Legacy version is showing the history, it ain't the best but it'll have to do for now until it gets fixed

**Comment by bigair43017** [comment]
I am having the same problem on all my firesticks. I tried loading an older version [12.50] with the same result. I assume youtube changed something. I hope there is a fix since I use history often.

**Comment by mathkkad** [comment]
UDPATE:

@Tailszefox While i have tested more in the last couple of hours i also found out the the latest beta 13.60 works fine. So i can also confirm that

Your error path is different because you were using a beta version. Mine is from the stable version
Also yes, this is a very recent problem, less than 24 hours old

Hope all our input info is useful and helps to solve the problem also for the stable version of the app

**Comment by DTK21** [comment]
Yep the 13.60 beta version definitely works. I thought they discontinued that and moved forward with the stable version

**Comment by Hank-Zhu** [comment]
I am having the same problem today on my MI box.

**Comment by sparkssssssss** [comment]
same issue,mark,waiting for solution.

**Comment by bigair43017** [comment]
I updated to the beta version on 5 firesticks yesterday. All worked fine. I had 1 to do today, but I decided to see if the old version was still not working. To my surprise it was working just fine today. It didn't work yesterday. I wonder what changed?

**Comment by yslupdates** [comment]
History not working for me either on v13.61. Using Onn 4K box

**Comment by Daydreamer00001** [comment]
Thanks for the tip. Manually updated to the new beta version to get my history back. Tried the stable version also and it wouldn't work at all with my YouTube login.
